src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +69,12 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(e -> e.getAge() >= fromAge
+                        && (e.getSex() == Person.Sex.MAN
+                        ? e.getAge() <= maleToAge
+                        : e.getAge() <= femaleToAge))
```

Comment:
this is too overcomplicated, let's just have it as follows
```suggestion
                .filter(e -> e.getAge() >= fromAge
                        && ((e.getSex() == Person.Sex.MAN && e.getAge() <= maleToAge)
                        || (e.getSex() == Person.Sex.WOMEN && e.getAge() <= femaleToAge)))
```